h/views/groups.py

```diff
@@ -58,6 +59,9 @@ def _js_config(self):
                 "link": self.request.route_url(
                     "group_read", pubid=group.pubid, slug=group.slug
                 ),
+                "num_annotations": self.annotation_stats_service.total_group_annotation_count(
+                    group.pubid, unshared=False
```

Comment:
is unshared always going to be False? Or are you thinking ahead of an immediate need for other type of group?